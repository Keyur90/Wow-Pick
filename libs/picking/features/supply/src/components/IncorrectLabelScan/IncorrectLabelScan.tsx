import React from 'react';
import { PopUpModal } from '@rf2/ui';

interface IncorrectLabelScan {
  isOpen: boolean;
  onClose: () => void;
}

const IncorrectLabelScan: React.FC<IncorrectLabelScan> = ({ isOpen, onClose }) => {
  return (
    <PopUpModal
      isOpen={isOpen}
      onOkClick={onClose}
      bodyText="Incorrect label scanned"
      headerText="Label Scan"
      closeOrOkText="Close"
    />
  );
};

export { IncorrectLabelScan };
