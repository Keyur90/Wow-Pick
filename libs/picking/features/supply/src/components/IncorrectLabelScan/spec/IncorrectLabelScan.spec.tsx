import React from 'react';
import { render } from '@testing-library/react';
import { IncorrectLabelScan } from '../IncorrectLabelScan';

test('renders IncorrectLabelScan message', () => {
  const { container } = render(<IncorrectLabelScan isOpen={true} onClose={null} />);

  expect(container).toMatchSnapshot();
});
