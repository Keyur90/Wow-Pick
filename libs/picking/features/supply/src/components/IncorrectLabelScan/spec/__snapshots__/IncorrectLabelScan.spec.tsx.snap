// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders IncorrectLabelScan message 1`] = `
<div
  aria-hidden="true"
/>
`;
